src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Program.cs

```diff
@@ -69,11 +70,18 @@ public static async Task MainAsync(string[] args)
                     .WithOutput(Console.OpenStandardOutput())
                     .ConfigureLogging(builder => builder
                         .AddLanguageServer()
-                        .SetMinimumLevel(logLevel))
+                        .SetMinimumLevel(RazorLSPOptions.GetLogLevelForTrace(trace)))
                     .OnInitialized(async (languageServer, request, response) =>
                     {
                         var fileChangeDetectorManager = languageServer.Services.GetRequiredService<RazorFileChangeDetectorManager>();
                         await fileChangeDetectorManager.InitializedAsync(languageServer);
+
+                        _ = Task.Delay(1000).ContinueWith(async task =>
+                        {
+                            // Adding a delay to workaround a bug (insert-link).
```

Comment:
(insert-link)

Reply:
I'll file a bug once I make sure this is one and then update this.